test/modules/TwinTester/ReportedPropertiesValidator.cs

```diff
@@ -52,14 +53,14 @@ public async Task ValidateAsync()
                 return;
             }
 
-            TwinCollection propertiesToRemoveFromTwin = await this.ValidatePropertiesFromTwinAsync(receivedTwin);
-            await this.RemovePropertiesFromStorage(propertiesToRemoveFromTwin);
-            await this.RemovePropertiesFromTwin(propertiesToRemoveFromTwin);
+            TwinCollection propertiesToRemove = await this.ValidatePropertiesFromTwinAsync(receivedTwin);
+            await this.RemovePropertiesFromStorage(propertiesToRemove);
+            await this.RemovePropertiesFromTwin(propertiesToRemove);
```

Comment:
If you want to consolidate these methods I think it is fine

Reply:
not for now. minimize the change. add a bug https://dev.azure.com/msazure/One/_workitems/edit/7272973